client/my-sites/site-settings/theme-enhancements.jsx

```diff
@@ -160,17 +195,14 @@ class ThemeEnhancements extends Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const { jetpackSettingsUI, translate } = this.props;
 		return (
 			<div>
 				<SectionHeader label={ translate( 'Theme Enhancements' ) } />
 
 				<Card className="theme-enhancements__card site-settings">
 					{ this.renderInfiniteScrollSettings() }
-
-					<hr />
-
-					{ this.renderMinilevenSettings() }
+					{ jetpackSettingsUI && <hr /> && this.renderMinilevenSettings() }
```

Comment:
Maybe it makes sense to add this to `renderJetpackInfiniteScrollSettings()` to avoid another `jetpackSettingsUI` check?